fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/mpjwtauth/MicroProfileJWTAuthFraction.java

```diff
@@ -1,17 +1,19 @@
-/**
- * Copyright 2017 Red Hat, Inc, and individual contributors.
+/*
```

Comment:
I think the preferred formatting is /** at the start as opposed to /*

Reply:
I'd personally argue that copyrights should start with `/*` rather than `/**`, as they are not really javadocs. But it seems that in Swarm, we typically use `/**`.